ui/src/app/pages/workspace/workspace-edit.spec.tsx

```diff
@@ -289,30 +290,31 @@ describe('WorkspaceEdit', () => {
     const wrapper = component();
     await waitOneTickAndUpdate(wrapper);
 
-    const accessTierSelection = wrapper.find('[data-test-id="select-access-tier"]');
+    const accessTierSelection = wrapper.find('[data-test-id="select-access-tier"]').find('select');
```

Comment:
so we're checking the actual select instead of the div with the data-test-id.  makes sense.